### ISSUE TITLE
[Bug]: Screenshot only captures upper left.

### ISSUE THREAD

**Issue by AltriaChan** [Floorp-Projects/Floorp#1149]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [ ] I have tried to create a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [X] I have included relevant screenshots or console outputs.
- [X] I have checked this problem is not ocurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

Screenshot only captures top left

### Reproduction Steps

1. right-click and select Take Screenshot. 
2. select the area you want to screenshot.
3. press the Download button.

### Expected Behavior

You'll see an image with the selection captured intact.

### Actual Behavior

The screenshot will capture only the upper left portion of the selected area.

### Screenshots

[https://andrewrreed-closed-vs-open-arena-elo.hf.space/?__theme=dark](https://andrewrreed-closed-vs-open-arena-elo.hf.space/?__theme=dark)
![스크린샷 2024-05-16 092624](https://github.com/Floorp-Projects/Floorp/assets/59331917/c07950f0-ab06-433a-86ce-63e9150e5df1)
![Screenshot 2024-05-16 at 09-26-33 Gradio](https://github.com/Floorp-Projects/Floorp/assets/59331917/efbbca64-8ce5-4999-835c-75ea538f468e)

### Browser Version

v11.12.2

### Operating System

Windows 11

### Additional Context

This has been an issue since I started using Floorp over a year ago, and I've never noticed it in Firefox.

**Comment by surapunoyousei** [comment]
![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/d3f997dd-ce99-4026-ae87-5ae3511f6adc)
It's no problem in my env

**Comment by DaiDucDaiUy** [comment]
Hello. I also had the same issue. Does anyone know how to fix it?

**Comment by creeper-0910** [comment]
It would be next to impossible to correct something that cannot be reproduced.
If you send us a pull request for this issue, we'll merge it in.

**Comment by Lampe2020** [comment]
I just installed Floorp yesterday from the official PPA on Ubuntu and noticed the screenshot button I have in my toolbar in Firefox wasn't available in the "edit toolbar" tab. I later found it in the "status bar", which I have hidden to make Floorp look as much like Firefox as possible, so I moved the button up to the tool bar, right next to the address bar.   
I noticed that neither it nor the "right-click → screenshot" option do anything at all.   
I have Floorp 11.14.1 on Ubuntu 22.04 LTS.   
By the way, installing Firefox's built-in screenshots extension doesn't make it work either.
